src/elements/Link/Link.tsx

```diff
@@ -3,9 +3,22 @@ import { color as styledColor, space, SpaceProps } from "styled-system"
 import { color } from "../../helpers"
 import { Color } from "../../Theme"
 
+enum UnderlineBehavior {
+  Default = "default",
+  Hover = "hover",
+  None = "none",
+}
+
+const computeUnderline = (state: string, behavior: UnderlineBehavior): string => {
+  const blocklist: UnderlineBehavior[] = state === "hover" ? [UnderlineBehavior.None] : [UnderlineBehavior.Hover, UnderlineBehavior.None]
+  const none = blocklist.includes(behavior)
```

Comment:
Wondering if Case statement here would be more descriptive of what are we trying to express. It took me a second to realize what  this `blocklist ` is blocking.